warehouse/query-core/src/main/java/datawave/query/planner/DefaultQueryPlanner.java

```diff
@@ -655,6 +658,11 @@ private void configureExcerpts(ShardQueryConfiguration config, IteratorSetting c
         }
     }
 
+    private void configureSummaries(ShardQueryConfiguration config, IteratorSetting cfg) {
```

Comment:
This should only be added if summaries are requested. Wrap in something like 
```if (config.getSummarySize() > 0) {```

Reply:
fixed